src/libraries/Microsoft.Extensions.Logging.Abstractions/src/Microsoft.Extensions.Logging.Abstractions.csproj

```diff
@@ -5,6 +5,8 @@
     <EnableDefaultItems>true</EnableDefaultItems>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPackable>true</IsPackable>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <ServicingVersion>1</ServicingVersion>
```

Comment:
From the PR description it's unclear to me why we ship new packages as part of this infrastructure release. Can you please explain?

Reply:
Hmm I see your concern. There's no actual code being modified, and there's only `gen` project files being edited.

So neither `Microsoft.Extensions.Logging.Abstractions.csproj` nor `System.Text.Json.csproj` need to be built for such changes?

Reply:
Since the generator projects ship in the package, I figured that we needed to update the package. And to update the package, the directions pointed to the src projects. I’m fine not publishing new packages, but I wanted to be on the safe side and follow the directions.

Reply:
No problem. Staying on the safe side is appreciated. Let's wait for Viktor's answer on this.

Reply:
> I’m fine not publishing new packages, but I wanted to be on the safe side and follow the directions.

Do the generator projects actually change because of the compiler upgrade in a customer visible way (i.e. perf, features, etc.)? If not, then I would rather not ship a new version of the packages.

Reply:
They have a different assembly reference in the metadata, but that would be it. I'll revert the packaging changes.